### PR TITLE
make useTLS true if the config prop is missing and fix docs

### PR DESCRIPTION
**Pull request by lmsurpre**
Signed-off-by: Lee Surprenant <lmsurpre@us.ibm.com>

### CODE REVIEW

File: fhir-server/src/main/java/com/ibm/fhir/server/listener/FHIRServletContextListener.java [hunk 1]
```diff
@@ -156,7 +156,7 @@ public void contextInitialized(ServletContextEvent event) {
 
                 // Gather up the NATS TLS properties.
                 Properties tlsProps = new Properties();
-                tlsProps.setProperty("useTLS", fhirConfig.getBooleanProperty(PROPERTY_NATS_TLS_ENABLED).toString());
+                tlsProps.setProperty("useTLS", fhirConfig.getBooleanProperty(PROPERTY_NATS_TLS_ENABLED, true).toString());
```
Comment (prb112) [inline fhir-server/src/main/java/com/ibm/fhir/server/listener/FHIRServletContextListener.java:159]:
```suggestion
                tlsProps.setProperty("useTLS", fhirConfig.getBooleanProperty(PROPERTY_NATS_TLS_ENABLED, Boolean.TRUE).toString());
```

Comment (prb112) [reply fhir-server/src/main/java/com/ibm/fhir/server/listener/FHIRServletContextListener.java:159]: pattern below is Boolean.True

Comment (lmsurpre) [reply fhir-server/src/main/java/com/ibm/fhir/server/listener/FHIRServletContextListener.java:159]: meh, can i change the below ones to "true" instead?

Comment (prb112) [reply fhir-server/src/main/java/com/ibm/fhir/server/listener/FHIRServletContextListener.java:159]: you'd have to change a bunch of spots in this class. I think this is consistent

**Pull request merged**